src/components/Publish/_constants.tsx

```diff
@@ -118,3 +118,6 @@ export const algorithmContainerPresets: MetadataAlgorithmContainer[] = [
 ]
 
 export const initialPublishFeedback: PublishFeedback = contentFeedback
+
+export const minAmountOcean = 50
```

Comment:
we declare them further down in the `initialValues`, no need to declare them here again. Just use the `initialValues` from the Formik context in any component you want